src/api/stats/getMultichainVaults.js

```diff
@@ -15,9 +16,13 @@ const REFRESH_INTERVAL = 5 * 60 * 1000;
 const LOG_PER_CHAIN = false;
 
 let vaultsByChain = {};
+let govVaultsByChain = {};
 let multichainVaults = [];
+let multichainGovVaults = [];
 let multichainVaultsCounter = 0;
+let multichainGovVaultsCounter = 0;
 let multichainActiveVaultsCounter = 0;
+let multichainGovActiveVaultsCounter = 0;
 let vaultsByID = {};
```

Comment:
I believe many of these fields are legacy ones that aren't needed. No need to have counters I'd say. It is useful to have the vaultsByID/govVaultsByID so I'd keep those. We could only keep `vaultsByChain, multichainVaults and vaultsByID` and their gov vault counterparts